protocol-units/bridge/cli/src/movement.rs

```diff
@@ -0,0 +1,141 @@
+use anyhow::Result;
+use bridge_shared::{
+	bridge_contracts::BridgeContractCounterparty,
+	types::{
+		Amount, BridgeTransferId, HashLock, HashLockPreImage, InitiatorAddress, RecipientAddress,
+		TimeLock,
+	},
+};
+use clap::{Parser, Subcommand};
+use movement_bridge::{utils::MovementAddress, MovementClient};
+
+#[derive(Parser)]
+#[command(name = "movement-cli")]
+#[command(about = "Movement Bridge client", long_about = None)]
+pub struct MovementCli {
+	#[command(subcommand)]
+	pub command: MovementCommands,
+}
+
+#[derive(Subcommand)]
+pub enum MovementCommands {
+	Deploy {
+		#[arg(short, long)]
+		config_path: String,
+	},
+	LockAssets {
+		#[arg(short, long)]
+		config_path: String,
+		#[arg(short, long)]
+		initiator: String,
+		#[arg(short, long)]
+		bridge_transfer_id: String,
+		#[arg(short, long)]
+		hash_lock: String,
+		#[arg(short, long)]
+		time_lock: u64,
+		#[arg(short, long)]
+		recipient: String,
+		#[arg(short, long)]
+		amount: u64,
+	},
+	Complete {
+		#[arg(short, long)]
+		config_path: String,
+		#[arg(short, long)]
+		bridge_transfer_id: String,
+		#[arg(short, long)]
+		pre_image: String,
+	},
+	Abort {
+		#[arg(short, long)]
+		config_path: String,
+		#[arg(short, long)]
+		bridge_transfer_id: String,
+	},
+	Details {
+		#[arg(short, long)]
+		config_path: String,
+		#[arg(short, long)]
+		bridge_transfer_id: String,
+	},
+}
+
+#[allow(unused)]
+pub async fn run(command: &MovementCommands) -> Result<()> {
+	match command {
+		MovementCommands::Deploy { config_path } => {
+			todo!()
+		}
+		MovementCommands::LockAssets {
+			config_path,
+			initiator,
+			bridge_transfer_id,
+			hash_lock,
+			time_lock,
+			recipient,
+			amount,
+		} => {
+			lock_assets(bridge_transfer_id, hash_lock, *time_lock, recipient, *amount).await?;
+		}
+		MovementCommands::Complete { config_path, bridge_transfer_id, pre_image } => {
```

Comment:
@0xmovses how is `config_path` used in these functions? It's not looking like it's passed in as an arg; would you pls explain the purpose of it? Is it the dot movement path?

Reply:
dot movement, and any suzuka config is, for now, completly un-related to the bridge. I added `config_path` so that later we could add some .yaml or .json files to set some config values, but for now this is just pre-emptive, so this param may be removed before the feature branch gets merged.